apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/db/ShardAddConstraintsSchemaVersion.java

```diff
@@ -55,7 +56,7 @@ protected int update(int nextUpdate) {
                 apply(null); //left for compatibility with existing dbs
             case startNumber + 11:
                 log.trace("Starting adding TRANSACTION constraint = {}", nextUpdate);
-                apply("alter table TRANSACTION add constraint IF NOT EXISTS PRIMARY_KEY_TRANSACTION_DB_ID primary key (DB_ID)"); // PK + unique index
+                apply("ALTER TABLE transaction ADD CONSTRAINT primary_key_transaction_db_id PRIMARY KEY (DB_ID)"); // PK + unique index
             case startNumber + 12:
```

Comment:
There is not exact convention written somewhere in our documents. 
But we will adhere to agreements from now like below:
- custom table/index/constraint names are LOWER CASE, because mariadb distinguish lower/upper case, it's case sensitive
- SQL keyword are better to keep in UPPER CASE